Expensify/App
# Use centered modal for errors in the VBA flow in NewDot

### Details


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/171666#

### Tests / QA
1. Click on the green + icon on the botton left and select new workspace if you dont already have one.
2. Click Get Started -> Connect manually
3. Use this [SO](https://stackoverflow.com/c/expensify/questions/342/525#525) and manually enter details.
4. When you see the company information screen enter incorrect values and click save and confirm.
5. Confirm you see the alert modal,
![image](https://user-images.githubusercontent.com/14356145/128581905-0f665556-c4f0-4968-8ff8-5117d850f387.png)

### QA Steps
1. Same as above. 

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/14356145/128581905-0f665556-c4f0-4968-8ff8-5117d850f387.png)

#### Mobile Web


#### Desktop


#### iOS
![image](https://user-images.githubusercontent.com/14356145/128583027-3657d8d4-5848-442a-8f29-edc220dff962.png)


#### Android
